libportal/spawn.c

```diff
@@ -131,6 +131,8 @@ do_spawn (SpawnCall *call)
 
   ensure_spawn_exited_connection (call->portal);
 
+  g_variant_builder_init (&opt_builder, G_VARIANT_TYPE ("a{sv}"));
```

Comment:
```suggestion
  g_variant_builder_init (&opt_builder, G_VARIANT_TYPE_VARDICT);
```
for consistency with the rest of the codebase

Reply:
fixed